client/src/main/java/se/fortnox/reactivewizard/client/RequestBuilder.java

```diff
@@ -15,6 +15,7 @@
  * is needed when the same http-proxy should use different hosts for different
  * calls.
  */
+@SuppressWarnings("checkstyle:MissingJavadocMethod")
```

Comment:
I had to add alot of SuppressWarnings since I couldn't really find how to specify the (updated) rules in order to pass checkstyle. They should obviously be removed before merging the PR.. I just had to make some progress with it :)

Reply:
Moved SuppressWarnings to methods instead of classes